minigalaxy/ui/properties.py

```diff
@@ -71,6 +75,16 @@ def ok_pressed(self, button):
         self.parent.parent.filter_library()
         self.destroy()
 
+    @Gtk.Template.Callback("on_button_properties_wine_file_set")
+    def on_button_properties_wine(self, widget):
+        self.game.set_info("custom_wine", str(self.button_properties_wine.get_filename()))
+
+    @Gtk.Template.Callback("on_button_properties_reset_clicked")
+    def on_menu_button_reset(self, widget):
+        if self.game.get_info("custom_wine") is not None:
```

Comment:
Wouldn't be enough?
if self.game.get_info("custom_wine"):

Reply:
It's enough, thanks :-)